server/helper.ts

```diff
@@ -269,6 +269,26 @@ function externalApiV2() {
     //Prepare body for ElasticSearch
     const bodyQuery = bodybuilder();
 
+    //Implementing Sorting Options
+    if (sortBy){
+      if (sortBy=="titleAscending")
+        bodyQuery.sort('titleStudy.raw', 'asc');
+      else if (sortBy=="titleDescending")
+        bodyQuery.sort('titleStudy.raw', 'desc');
+      else if (sortBy=="dateOfCollectionOldest")
+        bodyQuery.sort('dataCollectionPeriodEnddate', 'asc');
+      else if (sortBy=="dateOfCollectionNewest")
+        bodyQuery.sort('dataCollectionPeriodEnddate', 'desc');
+      else if (sortBy=="dateOfPublicationNewest")
+        bodyQuery.sort('publicationYear', 'desc');
+      else{
+        res.status(400).send({ message: 'Please provide a proper sorting option. Available: titleAscending, titleDescending, dateOfCollectionOldest, dateOfCollectionNewest, dateOfPublicationNewest'});
+        return;
+      }
```

Comment:
Considering the amount of conditionals, consider replacing the `if` statement with a `switch` statement.

```suggestion
    //Implementing Sorting Options
    switch (sortBy) {
      case "titleAscending":
        bodyQuery.sort('titleStudy.raw', 'asc');
        break;
      case "titleDescending":
        bodyQuery.sort('titleStudy.raw', 'desc');
        break;
      case "dateOfCollectionOldest":
        bodyQuery.sort('dataCollectionPeriodEnddate', 'asc');
        break;
      case "dateOfCollectionNewest":
        bodyQuery.sort('dataCollectionPeriodEnddate', 'desc');
        break;
      case "dateOfPublicationNewest":
        bodyQuery.sort('publicationYear', 'desc');
        break;
      default:
        res.status(400).send({ message: 'Please provide a proper sorting option. Available: titleAscending, titleDescending, dateOfCollectionOldest, dateOfCollectionNewest, dateOfPublicationNewest'});
        return;
    }
```